ci/render_pipelines.py

```diff
@@ -212,33 +212,40 @@ def mk_pipeline_promote_task(
 
 def mk_pipeline_notify_task(
     previous_tasks: typing.List[str],
-    gardenlinux_flavour: GardenlinuxFlavour,
-    all_tasks: typing.Sequence[tkn.model.Task]
+    all_tasks: typing.Sequence[tkn.model.Task],
+    build_tasks: typing.Sequence[tkn.model.Task],
+    platform_set: typing.Set[str] = frozenset(),
 ):
     status_dict = {}
+    build_dict = {}
     for task in previous_tasks:
         status_dict['status_' + task.name] = f'$(tasks.{task.name}.status)'
     status_str = json.dumps(status_dict)
 
-    if gardenlinux_flavour:
-        modifier_names = _get_modifier_names(gardenlinux_flavour)
-        platform = gardenlinux_flavour.platform
+    for task in build_tasks:
+        build_dict['build_' + task.name] = f'$(tasks.{task.name}.results.build_result)'
+    build_str = json.dumps(build_dict)
+
+    platform_set_str = ','.join(platform_set)
+
+    if build_tasks:
+        manifest_set_key = '$(tasks.promote-gardenlinux-task.results.manifest_set_key_result)'
     else:
-        modifier_names = ' '
-        platform = ' '
+        manifest_set_key = '~'
```

Comment:
marking just to make sure this isn't forgotten - also consider using `null`/`None`

Reply:
still not sure why isoformat should be better? Anyway I won't change this as it is already persisted in S3.

Reply:
~ changed to none